cf_remote/main.py

```diff
@@ -242,8 +245,24 @@ def _get_arg_parser():
         help="Append the new VMs to a pre-existing group",
         action="store_true",
     )
-    sp.add_argument("--aws", help="Spawn VMs in AWS (default)", action="store_true")
-    sp.add_argument("--gcp", help="Spawn VMs in GCP", action="store_true")
+    sp.add_argument(
+        "--provider",
+        help="VM provider",
+        type=str,
+        default="aws",
+        choices=["aws", "gcp", "vagrant"],
+    )
+    sp.add_argument("--cpus", help="Number of CPUs of the vagrant instances", type=int)
```

Comment:
Consider having a default 1 CPUs
```suggestion
    sp.add_argument("--cpus", default=1, help="Number of CPUs of the vagrant instances", type=int)
```

Reply:
I can't have a default cpu because I check if "cpus" is defined when validating args, because I don't allow to define the number of cpus for AWS spawned machines